codegen.js

```diff
@@ -30,7 +31,10 @@ const codegen = async () => {
   try {
     await fetchSchema();
 
-    const graphqlFiles = './src/graphql/**/*.graphql';
+    const graphqlFiles = [
+      './src/graphql/**/*.graphql',
+      './amplify/backend/api/colonycdapp/schema.graphql',
```

Comment:
Why do we need this file here? This array lists files that declare queries, not the schema itself

Reply:
you are completely correct this was a mistake on my end as I was trying to get changes to the schema to be automatically picked up. I have now removed this extra file and discovered that it is possible to regenerated the `generated.ts` file if we run codegen in a separate terminal (and then close it down) *after* the amplify container has correctly generated a new schema from the changes. This could be done nicer but I think it is good enough for now

Reply:
I don't think there's much value in running codegen together with amplify as you're most likely to need it when you change the schema, and as you've noticed that requires running it separately anyway. I'll raise this at the dev call next week